aptos-move/aptos-transactional-test-harness/tests/diem_test_harness/example.exp

```diff
@@ -1,17 +1,26 @@
 processed 10 tasks
 
+task 2 'publish'. lines 19-30:
+Error: Transaction discarded. VMStatus: status INVALID_AUTH_KEY of type Validation
```

Comment:
I think those are unexpected

Reply:
After debugging, I concatenate the auth_prefix + account_address to generate the new account address to make these tests work in 32-byte address. will followup to update the move code to completely remove the authkey_prefix in a separate PR